### PR TITLE
removing unnecessary code

### PR DESCRIPTION
**Pull request by JeanLoureiro**

### CODE REVIEW

**Review by goneale — CHANGES_REQUESTED** [review]

File: src/Accounts/Accounts.tsx [hunk 1]
```diff
@@ -8,28 +8,50 @@ import {
...
+    setSelectedUser(item);
+  };
+
+  const handleDelete = () => {
+    deleteAccount(host, token, selectedUser.id).subscribe(
+      () => {
+        fetchData();
+        setDeleteDialog(false);
+      },
+      (error: any) => console.log(error),
```
[the remark this answers is not in the document]
Comment (JeanLoureiro) [reply src/Accounts/Accounts.tsx:60]: removed

File: src/DataServices/DataServices.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import { identity } from 'lodash';
```
[the remark this answers is not in the document]
Comment (JeanLoureiro) [reply src/DataServices/DataServices.ts:1]: removed

File: src/Metadata/Metadata.tsx [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+  Select,
+  Switch,
+  TextField,
+  Typography,
+  Box,
+  Chip,
+} from '@material-ui/core';
+import { Autocomplete } from '@material-ui/lab';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import { onErrorResumeNext } from 'rxjs/operators';
```
[the remark this answers is not in the document]
Comment (JeanLoureiro) [reply src/Metadata/Metadata.tsx]: removed

File: src/Table/types.ts [hunk 4]
```diff
@@ -0,0 +1,36 @@
+interface DefaultTableProps {
+  error: boolean;
+  loading: boolean;
+  tableHeaders: TableHeaders[];
+  data: TableData[];
+  searchItems: (item: any) => void;
```
[the remark this answers is not in the document]
Comment (JeanLoureiro) [reply src/Table/types.ts]: only left `Record<any, any>` and `searchItems: (item: any) => void;`

File: src/UserGroups/UserGroupForm.tsx [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+      }
+    });
+  };
+
+  const handleSubmit = (e: MouseEvent<HTMLButtonElement>) => {
+    e.preventDefault();
+    onSubmit(form);
+  };
+
+  const handleError = (boolean) => {
```
[the remark this answers is not in the document]
Comment (JeanLoureiro) [reply src/UserGroups/UserGroupForm.tsx:96]: done

File: src/UserGroups/UserGroups.tsx [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+    setSelectedUserGroup(item);
+  };
+
+  const handleDelete = () => {
+    deleteUserGroup(host, token, selectedUserGroup.id).subscribe(
+      () => {
+        fetchData();
+        setDeleteDialog(false);
+      },
+      (error: any) => console.log(error),
```
[the remark this answers is not in the document]
Comment (JeanLoureiro) [reply src/UserGroups/UserGroups.tsx:59]: done,

File: src/UserGroups/type.ts [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+  token: string;
+  metadata?: Metadata[];
+  translations?: {
+    /* not impl */
+  };
+}
+
+interface UserGroupTableHeaderProps {
+  filter: string;
+  setFilter: (event: any) => any;
```
[the remark this answers is not in the document]
Comment (JeanLoureiro) [reply src/UserGroups/type.ts]: fixed

File: src/DataServices/DataServices.ts [hunk 8]
```diff
@@ -111,19 +112,54 @@ const fetchTimeseriesByGroup = (dataSources: DataSource[], token: string) => {
...
   fetchUrl(`${host}/api/usergroups/user/${data.userId}`, {
     method: 'POST',
     additionalHeaders: { Authorization: `Bearer ${token}` },
     body: JSON.stringify(data.groups),
   }).pipe(tap((res) => console.log('fetchUserGroups', res)));
 
+const createUserGroup = (
+  host: string,
+  token: string,
+  data: { id: string; name: string; users: string[]; metadata: {} },
```
[the remark this answers is not in the document]
Comment (JeanLoureiro) [reply src/DataServices/DataServices.ts:131]: added `UserGroupsData` instead

**Pull request merged**